manifests/init.pp

```diff
@@ -173,9 +164,6 @@
 # $dhcp::                       Enable DHCP feature
 #                               type:boolean
 #
-# $dhcp_split_config_files::    Split DHCP configuration files. This is needed since version 1.11.
-#                               type:boolean
-#
```

Comment:
Well spotted, thanks! It should have been removed with the split config files, as it's 1.10 and older. Updated.